src/applications/vaos/services/appointment/index.js

```diff
@@ -910,6 +910,27 @@ export async function cancelAppointment({ appointment, useV2 = false }) {
   return cancelRequestedAppointment(appointment);
 }
 
+/**
+ * Get the first practitioner name from appointment object
+ *
+ * @export
+ * @param {Object} appointment an appointment object
+ * @returns {String} Returns the provider first and last name
+ */
+export function getProviderName(appointment) {
+  const providers = appointment?.practitioners;
+  if (
+    appointment.practitioners !== undefined &&
+    providers[0]?.name !== undefined &&
+    providers[0]?.name?.family !== undefined
+  ) {
+    const lastName = providers[0]?.name?.family;
+    const firstName = providers[0]?.name?.given[0];
+    return `${firstName} ${lastName}`;
+  }
+  return null;
+}
```

Comment:
Here's another implementation
```
  // Object destructuring
  const { practitioners } = appointment;
  // Array destructuring with a default empty array when practitioners is null/undefined
  // We only grab the 1st element
  const [practitioner] = practitioners || [];
  // Object destructuring with a default empty object when practitioners is null/undefined
  const { name } = practitioner || {};

  return name ? `${name.family} ${name.given[0]}` : null;
  ```

Reply:
I like your object destructuring. Makes for much cleaner code.